file_transfer_protocol/client.py

```diff
@@ -1,13 +1,13 @@
 import socket                   # Import socket module
 
-s = socket.socket()             # Create a socket object
+sock = socket.socket()             # Create a socket object
 host = socket.gethostname()     # Get local machine name
 port = 12312
 
-s.connect((host, port))
-s.send(b'Hello server!')
+sock.connect((host, port))
+sock.send(b'Hello server!')
 
-with open('Received_file', 'wb') as f:
+with open('Received_file', 'wb') as file:
```

Comment:
Please change __file__ to __out_file__ to make the filemode clear to the reader.